src/logger/index.js

```diff
@@ -5,21 +5,49 @@ function loggerFactory() {
   let logger
 
   function create() {
+    let logger
+
+    if (globalEnv.isBrowser) {
+      // on client side,
+      // we use console directly
+      // TODO:
+      // send client side log to stackdriver logging
+      logger = console
+
+      // since we use the same logger both on client and server side,
+      // we mock `errorReport` function here.
+      logger.errorReport = console.error
+      return logger
+    }
+
+
     // on server side,
     // we use winston and logging-winston to log
-    if (!globalEnv.isBrowser) {
+    if (globalEnv.isProduction) {
+      logger = createProdLogger()
+    }
+    logger = createDevLogger()
```

Comment:
Create ProdLogger() if isProduction is true?

Reply:
It's a bug.
updated.